walkthroughs/howto-access-log/README.md

```diff
@@ -0,0 +1,174 @@
+   If you do not already have one, you can create a keypair using the command below if you don't have one. See [Amazon EC2 Key Pairs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html).
+
+    ```bash
+    aws ec2 create-key-pair --key-name color-app | jq -r .KeyMaterial > ~/.ssh/color-app.pem
+    chmod 400 ~/.ssh/color-app.pem
+    ```
+
+    This command creates an Amazon EC2 Key Pair with name `color-app` and saves the private key at `~/.ssh/color-app.pem`.
+
+4. Additionally, this walkthrough makes use of the unix command line utility `jq`. If you don't already have it, you can install it from [here](https://stedolan.github.io/jq/).
+5. Install Docker. It is needed to build the demo application images.
+
+![](fargate.png)
+## Step 2: Set Environment Variables and deploy color app
+We need to set a few environment variables before provisioning the infrastructure.
+Please set the value for `AWS_ACCOUNT_ID`, `KEY_PAIR_NAME`,'AWS_DEFAULT_REGION' and `ENVOY_IMAGE` below.
+
+```bash
+export AWS_ACCOUNT_ID=<your account id>
+export KEY_PAIR_NAME=<color-app or your SSH key pair stored in AWS>
+export AWS_DEFAULT_REGION=<your region here>
+export ENVOY_IMAGE=840364872350.dkr.ecr.<region-code>.amazonaws.com/aws-appmesh-envoy:v1.22.2.1-prod
+export AWS_PROFILE=default
+export ENVIRONMENT_NAME=LOGGING
+export MESH_NAME=appmesh-mesh-logging
+export SERVICES_DOMAIN=logging.local
+export GOPROXY=direct
+export GO_PROXY=direct
+```
+This example is based on color app with Virtual Node on both EC2 and Fargate instance. Follow the color app example in aws-app-mesh-examples/examples/apps/colorapp/
+then follow walkthrough in aws-app-mesh-examples/walkthroughs/fargate/ to the step where you have 2 active virtual nodes blue (on EC2) and green (on fargate).
+
+Or you can follow the following steps:
+
+We'll start by setting up the basic infrastructure for our services.
+
+1. create the VPC.
+
+```bash
+./infrastructure/vpc.sh
+```
+2. Create the Mesh
+
+```bash
+./infrastructure/appmesh-mesh.sh
+```
+3. create the ECS cluster
+
+```bash
+./infrastructure/ecs-cluster.sh
+```
+4. create mesh resources
+
+```bash
+./colorapp-ecs/appmesh-colorapp.sh
+```
+5. Deploy service to ECS
+
+```bash
+aws ecr create-repository --repository-name=gateway
+export COLOR_GATEWAY_IMAGE=$(aws ecr describe-repositories --repository-names=gateway --query 'repositories[0].repositoryUri' --output text)
+./colorapp-ecs/gateway/deploy.sh
+```
+
+```bash
+aws ecr create-repository --repository-name=colorteller
+export COLOR_TELLER_IMAGE=$(aws ecr describe-repositories --repository-names=colorteller --query 'repositories[0].repositoryUri' --output text)
+./colorapp-ecs/colorteller/deploy.sh
+```
+
+```bash
+./colorapp-ecs/ecs-colorapp.sh
+```
+
+To test:
+
+```bash
+colorapp=$(aws cloudformation describe-stacks --region=$AWS_DEFAULT_REGION --stack-name=$ENVIRONMENT_NAME-ecs-colorapp --query="Stacks[0].Outputs[?OutputKey=='ColorAppEndpoint'].OutputValue" --output=text)
```

Comment:
nit: I don't think there's any need for --region=$AWS_DEFAULT_REGION on aws cli commands. I think that var takes effect automatically. https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html